packages/material-ui/src/Modal/Modal.test.js

```diff
@@ -11,8 +11,7 @@ import {
   createMount,
   describeConformanceV5,
 } from 'test/utils';
-import { createMuiTheme } from '@material-ui/core/styles';
-import { ThemeProvider } from '@material-ui/styles';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
```

Comment:
Now that we have a ModalUnstyled, it makes sense (the default theme change doesn't matter)